resources/control_systems.md

```diff
@@ -5,66 +5,712 @@
+    - [5.4 Matlab Implementation](#54-matlab-implementation)
+- [6 Time Response Analysis](#6-time-response-analysis)
+    - [6.1 Transient State Response](#61-transient-state-response)
+    - [6.2 Steady State Response](#62-steady-state-response)
+    - [6.3 Standard Test Signal](#63-standard-test-signal)
+        - [6.3.1 Unit Impulse Signal](#631-unit-impulse-signal)
+        - [6.3.2 Unit Step Signal](#632-unit-step-signal)
+        - [6.3.3 Unit Ramp Signal](#633-unit-ramp-signal)
+- [7 Response of a First Order System](#7-response-of-a-first-order-system)
+- [8 Response of a Second Order System](#8-response-of-a-second-order-system)
+- [9 Time Domain Specifications of a 2nd Order System](#9-time-domain-specifications-of-a-2nd-order-system)
+    - [9.1 Delay Time](#91-delay-time)
+    - [9.2 Rise Time](#92-rise-time)
+    - [9.3 Peak Time](#93-peak-time)
+    - [9.4 Peak Overshoot](#94-peak-overshoot)
+    - [9.5 Settling Time](#95-settling-time)
+- [10 Steady State Error](#10-steady-state-error)
+- [11 Stability](#11-stability)
+    - [11.1 Types of Stability](#111-types-of-stability)
+- [12 Frequency Response](#12-frequency-response)
+- [13 Frequency Domain Specifications](#13-frequency-domain-specifications)
+    - [13.1 Resonant Frequency](#131-resonant-frequency)
+    - [13.2 Peak Resonance](#132-peak-resonance)
+    - [13.3 Bandwidth](#133-bandwidth)
+- [14 Control System Plots](#14-control-system-plots)
+    - [14.1 Bode Plot](#141-bode-plot)
+        - [14.1.1 Stability Analysis with Bode Plots](#1411-stability-analysis-with-bode-plots)
+            - [14.1.1.1 Cross Over Frequency](#14111-cross-over-frequency)
+            - [14.1.1.2 Gain and Phase Margin](#14112-gain-and-phase-margin)
+        - [14.1.2 Matlab Implementation](#1412-matlab-implementation)
+    - [14.2 Nyquist Plot](#142-nyquist-plot)
+        - [14.2.1 Nyquist Stability Criterion](#1421-nyquist-stability-criterion)
+        - [14.2.2 Stability Analysis with Nyquist Plots](#1422-stability-analysis-with-nyquist-plots)
+            - [14.2.1.1 Cross Over Frequency](#14211-cross-over-frequency)
+            - [14.2.1.2 Gain and Phase Margin](#14212-gain-and-phase-margin)
+        - [14.2.3 Matlab Implementation](#1423-matlab-implementation)
+- [15 State Space](#15-state-space)
+    - [15.1 State Transition Matrix](#151-state-transition-matrix)
+    - [15.2 Controllability and Observability](#152-controllability-and-observability)
+- [16 Code Implementation](#16-code-implementations)
 
 </details>
 
 ---
 
 ## 1 Intro to Control Systems
 
-> Recommended video:
+Control systems constitute a system, device or set of devices, that utilizes the principle of output control via input,
+to command the activities of other systems or devices.  
+Control systems are utilized in electronic, automation and engineering systems.
+
+**Recommended video:**
+
 > [Everything You Need to Know About Control Theory](https://youtu.be/lBC1nEq0_nk)
```

Comment:
This is very minor and picky, but try to keep consistent. Throughout the file there are generally 3 different formats used:

1. With line break and quote only hyperlink.
  ```markdown
  **Recommended video:**
  
  > [GitHub](https://github.com)
  ```
2. No line break and quote only hyperlink.
  ```markdown
  **Recommended video:**
  > [GitHub](https://github.com)
  ```
3. No line break and quote all.
  ```markdown
  > **Recommended video:**
  > [GitHub](https://github.com)
  ```

I personally follow something most similar to the 3rd format. You can pick. I recommend you learn how to use the refactoring tools or find / replace tools. I would use [Find & Select Multiple Occurrences](https://www.jetbrains.com/guide/java/tips/find-select-multiple-occurrences/) feature in JetBrains IDEs.